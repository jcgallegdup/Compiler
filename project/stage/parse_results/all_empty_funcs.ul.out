Insert key=Implement me! val=AST.Function@76ed5528
Insert key=Implement me! val=AST.Function@2c7b84de
Insert key=Implement me! val=AST.Function@3fee733d
Insert key=Implement me! val=AST.Function@5acf9800
Insert key=Implement me! val=AST.Function@4617c264
Insert key=Implement me! val=AST.Function@36baf30c
Insert key=Implement me! val=AST.Function@7a81197d
Insert key=Implement me! val=AST.Function@5ca881b5
Insert key=Implement me! val=AST.Function@24d46ca6
Insert key=Implement me! val=AST.Function@4517d9a3
Insert key=Implement me! val=AST.Function@372f7a8d
Insert key=Implement me! val=AST.Function@2f92e0f4
Insert key=Implement me! val=AST.Function@28a418fc
Insert key=Implement me! val=AST.Function@5305068a
Insert key=Implement me! val=AST.Function@1f32e575
Insert key=Implement me! val=AST.Function@279f2327
Insert key=Implement me! val=AST.Function@2ff4acd0
Insert key=Implement me! val=AST.Function@54bedef2
Insert key=Implement me! val=AST.Function@5caf905d
Insert key=Implement me! val=AST.Function@27716f4
Insert key=Implement me! val=AST.Function@8efb846
Error:0(line):0(position): Could not find function with name 'main' and type 'void'.
