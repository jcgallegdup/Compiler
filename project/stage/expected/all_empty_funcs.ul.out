boolean _kK1_jJ2 ()
{
}

char fF1_jJ2_ ()
{
}

float _0fF_9zZ ()
{
}

int[100] K1k_J2j_ ()
{
}

int _____ ()
{
}

string lowercase ()
{
}

void UPPERCASE ()
{
}

void FO_1f23___ (int FO_1f23___)
{
}

char _FFFFffff1234 (char _FFFFffff1234)
{
}

float _0_123456789 (boolean as_2394dkKJD)
{
}

int[100] _123456_FooZ (float _____1)
{
}

int _____123567FkoP (string __123Kk)
{
}

string f12345098_234fXx_ (void UPPERCASE)
{
}

void f12345098_234fXx_ (int[234] lowercase)
{
}

void fO_123456789_Xa (int asdf_kdf23_, int[89] MEOW_meow_1094385, int[1010] _meow_1094385, int[8912] _1094385)
{
}

char f12345098_234fXx_ (char lowercase, float UPPERCASE, boolean ______, void _1234567890)
{
}

float _O_123456789 (boolean AAas_2394dkKJD, float __Ka235, string _KDJ___123, float __Ka235, string _KD1J___123, float __Ka235, string _KDJ___123)
{
}

int[100] _123456_FooZ (float _____1, void __KDkk1236, string __3426Kka)
{
}

int _____123567FkoP (string __123Kk, string __234IKk, string __234IKk, string __234IKk, int UPPERCASE)
{
}

string f12345098_234fXx_ (void askj_234wf, int D0_1234Kkk)
{
}

void f12345098_234fXx_ (int[234] _234JKDd, int ___1Kk, string _123KkksldjbB_)
{
}
